### ISSUE TITLE
[Bug] MBPO loss explode on humanoid and walker (sometimes)

### ISSUE THREAD

**Issue by MansonX** [facebookresearch/mbrl-lib#137]
## Steps to reproduce

  1. Install latest mbrl-lib with python 3.7 pytorch 1.7.1 mujoco 2.0.2.13
  2. On humanoid: python -m mbrl.examples.main algorithm=mbpo overrides=mbpo_humanoid dynamics_model.hid_size=400
  >  On walker (sometimes): python -m mbrl.examples.main algorithm=mbpo overrides=mbpo_walker 

## Observed Results

+ on humanoid:
> I change some cfgs, but loss explode on humanoid always happens, just like what you said in you paper :-(
| results | S: 999 | R: 305.9237 | E: 0 | RL: 1
| train | S: 1000 | BR: 4.7887 | ALOSS: -6.8791 | CLOSS: 2.3326 | TLOSS: 2.2119 | TVAL: 0.1731 | AENT: 10.7918
| train | S: 2000 | BR: 4.7896 | ALOSS: -8.1911 | CLOSS: 6.5361 | TLOSS: 1.1089 | TVAL: 0.1353 | AENT: 6.0317
| train | S: 3000 | BR: 4.7901 | ALOSS: -28.7093 | CLOSS: 235.4853 | TLOSS: -1.3383 | TVAL: 0.1363 | AENT: -11.1760
| train | S: 4000 | BR: 4.7891 | ALOSS: -171.7771 | CLOSS: 10692.9198 | TLOSS: -7.0062 | TVAL: 0.2216 | AENT: -33.1193
| train | S: 5000 | BR: 4.7898 | ALOSS: -778.7793 | CLOSS: 251870.5247 | TLOSS: -16.1013 | TVAL: 0.3613 | AENT: -45.4896
| train | S: 6000 | BR: 4.7901 | ALOSS: -2556.9596 | CLOSS: 3338786.4839 | TLOSS: -31.5729 | TVAL: 0.5803 | AENT: -56.1798
| train | S: 7000 | BR: 4.7895 | ALOSS: -7508.6594 | CLOSS: 26147583.6990 | TLOSS: -62.3435 | TVAL: 0.9089 | AENT: -69.8062

+ on walker: (not always)
| train | S: 2061000 | BR: 0.9034 | ALOSS: -57449213.0880 | CLOSS: 30649331655639.0391 | TLOSS: 34.0375 | TVAL: 92662.5002 | AENT: -3.0001
| train | S: 2062000 | BR: 0.9086 | ALOSS: -57937056.9280 | CLOSS: 30568978606718.9766 | TLOSS: -794.3958 | TVAL: 95031.0067 | AENT: -3.0092
| train | S: 2063000 | BR: 0.9088 | ALOSS: -58598286.2720 | CLOSS: 30982745023840.2578 | TLOSS: -1195.4651 | TVAL: 98324.6478 | AENT: -3.0133
| train | S: 2064000 | BR: 0.9027 | ALOSS: -58989812.7840 | CLOSS: 32028101398495.2305 | TLOSS: 325.0210 | TVAL: 99747.0493 | AENT: -2.9972
| train | S: 2065000 | BR: 0.9084 | ALOSS: -59708167.0400 | CLOSS: 33024967268368.3828 | TLOSS: 47.7344 | TVAL: 99133.9851 | AENT: -3.0003

## Expected Results

Just like the result on cheetah...

**Comment by luisenp** [comment]
Hi @MansonX. Unfortunately, we haven't been able to get MBPO working on Humanoid yet, as mentioned in our white [paper](https://arxiv.org/pdf/2104.10159.pdf). For walker, do you have a sense of how often this issue appears? In my runs, the hyperparameters in the repo (which I just double checked) consistently work for 10 seeds, although I just noticed that I have only run it with `+overrides.bootstrap_permutes=true`, but I'm not sure how much of an effect this will have in training.

**Comment by MansonX** [comment]
I will run a few more experiments on walker with ```+overrides.bootstrap_permutes=true``` and report if that issue appears again :-(     
thanks and good luck!

**Comment by chenxi-yang** [comment]
@MansonX May I ask if you got the luck to make walker2d work? Separately, I also have issues with hopper. Did you get Hopper-v2 work?
Thanks!

**Comment by MansonX** [comment]
> 

My MBPO works well on both env you mentioned after SAC get changed. You can check this issue. https://github.com/facebookresearch/mbrl-lib/issues/138.
